polar-core/src/error.rs

```diff
@@ -350,6 +359,18 @@ The expression is: {expr}
                 );
                 write!(f, "{}", msg)
             }
+            Self::DataFilteringUnsupportedOp { operation } => {
+                let msg = formatdoc!(
+                    r#"Unsupported operation: {}
+
+                    This operation is not supported for data filtering.
+                    For more information please refer to our documentation:
+                        https://docs.osohq.com/guides/data_filtering.html
+                    "#,
+                    operation.to_polar()
```

Comment:
Does the `Display` impl for `Operation` not default to calling `.to_polar()`?

Reply:
huh, apparently we don't have a `Display` impl for `Operation` — we should add that